src/resin-event-log.js

```diff
@@ -87,7 +87,8 @@ var EVENTS = {
 	billing: [ 'paymentInfoUpdate', 'planChange', 'invoiceDownload' ],
 	onboarding: [ 'stepClick', 'whatNextItemClick'],
 	page: [ 'visit' ],
-	navigation: ['click']
+	navigation: ['click'],
+	members: [ 'create', 'edit', 'delete', 'invite' ],
```

Comment:
Not sure whether it would make more sense if we did:
```suggestion
	members: [ 'add', 'edit', 'delete', 'invite' ],
```

Reply:
@thgreasi the others were `create`, so I just wanted to keep the same terminology for CRUD actions.